backend/wmg/data/cube_pipeline.py

```diff
@@ -30,9 +46,11 @@ def load_data_and_create_cube(path_to_datasets: str, corpus_name: str, log_level
     A per tissue mapping of cell ontologies is generated and the files are copied to s3 under a shared timestamp,.
     On success the least recent set of files are removed from s3
     """
-    s3_uris = get_s3_uris()
-    copy_datasets_to_instance(s3_uris, path_to_datasets)
-    load(path_to_datasets, corpus_name)
+    if not tiledb.VFS().is_dir(corpus_name):
+        create_tdb(corpus_name)
+    # s3_uris = get_s3_uris()
```

Comment:
currently commented out for ease of testing, will comment back in before merging
same with line below